### PR TITLE
Filters and query params on the URLs

### PR DESCRIPTION
**Pull request by sanbornsen**

### CODE REVIEW

**Comment by fabric8cd** [comment]
@sanbornsen snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1688-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@sanbornsen snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1688-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/work-item/work-item-board/work-item-board.component.html [hunk 1]
```diff
@@ -35,7 +38,7 @@
                     </div>
                   </div>
                   <div [dragula]='"first-bag"' class="card-wrapper">
-                    <div class="board-card card-pf card-pf-view card-pf-view-select card-pf-view-single-select" [class.active]="isSelected(item)" *ngFor='let item of lane.workItems' routerLink="./detail/{{item.id}}" [attr.data-id]="item.id"
+                    <div class="board-card flex-container in-column-direction flex-grow-1" *ngFor='let item of lane.workItems' [attr.data-id]="item.id"
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-board/work-item-board.component.html]: 1. please do not add flex
2. why were the selected classes removed?

**Comment by mindreeper2420** [comment]
@vikram-raj @sanbornsen why was flex added to line 41 of work-item-board.component.html and why were the card selection classes removed?

File: src/app/work-item/work-item-board/work-item-board.component.html (hunk 1 above)
Comment (sanbornsen) [inline src/app/work-item/work-item-board/work-item-board.component.html]: Sorry @mindreeper2420 , that's clearly a rebase mistake. I'll fix that.

Comment (sanbornsen) [inline src/app/work-item/work-item-board/work-item-board.component.html]: @mindreeper2420 here you go - https://github.com/fabric8io/fabric8-planner/pull/1692 let me know if you find anything else.

**Pull request merged**